lib/turn_junebug_expressway_web/utils.ex

```diff
@@ -19,4 +23,44 @@ defmodule TurnJunebugExpresswayWeb.Utils do
       {_, true} -> {:ok, "good hmac"}
     end
   end
+
+  def format_message(conn) do
+    {:ok, body} = Jason.decode(conn.private[:raw_body])
+
+    today = DateTime.utc_now()
+
+    {:ok,
+     %{
+       "content" => get_in(body, ["text", "body"]),
+       "message_version" => "20110921",
+       "message_type" => "user_message",
+       "to_addr" => get_in(body, ["to"]),
+       "from_addr" => get_env(:junebug, :from_addr),
+       "message_id" => Ecto.UUID.generate(),
+       "timestamp" => DateTime.to_string(today),
+       "in_reply_to" => nil,
+       "session_event" => nil,
+       "transport_name" => get_env(:junebug, :transport_name),
+       "transport_type" => get_env(:junebug, :transport_type),
+       "transport_metadata" => %{}
+     }}
+  end
+
+  def send_message(message) do
+    options = [
+      host: get_env(:rabbitmq, :host),
+      port: get_env(:rabbitmq, :port),
+      virtual_host: get_env(:rabbitmq, :vhost),
+      username: get_env(:rabbitmq, :user),
+      password: get_env(:rabbitmq, :password)
+    ]
+
+    {:ok, connection} = AMQP.Connection.open(options, :undefined)
```

Comment:
Probably not for this PR, but I'm wondering if we can have a genserver or something running that keeps the connection open, and then sends messages over that connection, to avoid opening an closing a connection for each message.